### PR TITLE
Add command line to programming workflow slides

### PR DESCRIPTION
**Pull request by hharen**

### CODE REVIEW

File: slides/programming-workflow/index.html [hunk 1]
```diff
@@ -64,7 +64,103 @@ <h2>Programming workflow</h2>
           </section>
 
           <section>
-            TODO: Explain terminal: cd, ls, tab completion, arrow keys (history)
+            <h2>Command line</h2>
+            <img class="no-border" src="images/terminal.png" style="max-height: 500px">
```
Comment (nerdinand) [inline slides/programming-workflow/index.html]: this image doesn't exist?

File: slides/programming-workflow/index.html [hunk 2]
```diff
@@ -64,7 +64,103 @@ <h2>Programming workflow</h2>
...
+            </p>
+            <p class="fragment fade-in">
+              Cryptic commands might seem scary at first
+            </p>
+          </section>
+
+          <section>
+            <h2>Basic command line commands</h2>
+            <p>
+              <b>pwd</b> stands print working directory
```
Comment (nerdinand) [inline slides/programming-workflow/index.html]: I'd probably use `<pre class="pre-inline">pwd</pre>` for the command examples. It's more consistent with the rest of the slides that way. What do you think?

Screenshot 2020-01-14 at 21 18 20

Comment (nerdinand) [reply slides/programming-workflow/index.html]: (dito for the other places where there are shell commands)

Comment (hharen) [reply slides/programming-workflow/index.html]: I think that the red color stands out more but you're right with the consistency :P

File: slides/programming-workflow/index.html (hunk 1 above)
Comment (hharen) [reply slides/programming-workflow/index.html]: added

File: slides/programming-workflow/index.html (hunk 2 above)
Comment (hharen) [reply slides/programming-workflow/index.html]: ok

**Pull request merged**